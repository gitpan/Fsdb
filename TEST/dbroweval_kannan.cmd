prog='dbroweval'
args="-f TEST/dbroweval_kannan.code"
cmp='diff -cb '
