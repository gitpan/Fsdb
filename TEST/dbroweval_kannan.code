_sum = 0;
foreach (0..1) { _sum += _test1($_); }
