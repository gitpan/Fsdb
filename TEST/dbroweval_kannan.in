#h name id test1 test2 sum
a       1       80      81      0
b       2       70      71      0
c       3       65      66      0
d       4       90      91      0
e       5       70      71      0
f       6       90      91      0
#  | dbcolcreate -d sum 0
