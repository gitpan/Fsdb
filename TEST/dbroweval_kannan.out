#fsdb name id test1 test2 sum
a	1	80	81	161
b	2	70	71	141
c	3	65	66	131
d	4	90	91	181
e	5	70	71	141
f	6	90	91	181
#  | dbcolcreate -d sum 0
#   | dbroweval   { _sum = 0; foreach (0..1) { _sum += _test1($_); } ; }
